plugins/jobs/web_client/JobStatus.js

```diff
@@ -47,12 +53,20 @@ var JobStatus = {
     registerStatus: function (status) {
         _.each(status, function (info, name) {
             this[name] = info.value;
-            this._map[info.value] = {
+
+            let statusInfo = {
                 text: info.text,
                 icon: info.icon,
                 color: info.color,
+                textColor: 'white',
                 finished: _.has(info, 'finished') ? info.finished : false
             };
+
+            if ('textColor' in info) {
```

Comment:
Yeah, it's not a big deal in this context, but IMO, given that it now raises a mental red flag for me and other Crockford disciples, it decreases the readability of the code every time I see `in` used, because I have to stop and think about whether it's intentional or a bug.